_stbt/match.py

```diff
@@ -683,8 +684,8 @@ def _match_template(image, template, mask, method, roi_mask, level, imwrite):
             cv2.rectangle(
                 source_with_rois,
                 (max(0, r.x), max(0, r.y)),
-                (min(s.w - 1, r.x + r.w + t.w - 1),
-                 min(s.h - 1, r.y + r.h + t.h - 1)),
+                (min(s.w - 1, r.right + t.w - 1),
+                 min(s.h - 1, r.bottom + t.h - 1)),
```

Comment:
Right, this is the same as:

```python
d = roi.extend(right=template.shape[1], bottom=template.shape[0])
d = Region.intersect(d, _image_region(source_with_rois))
cv2.rectangle(source_with_rois, (d.x, d.y), (d.right - 1, d.bottom - 1),
              (0, 255, 255), thickness=1)
```

Maybe we should have our own draw_rectangle function that takes a `Region` encompassing the last two lines.